src/resource/resource.ts

```diff
@@ -181,12 +181,18 @@ export function isPodOwner(
  * @since 1.2.0
  */
 export class FetchError extends SolidClientError {
-  public readonly statusCode: number;
-  public readonly statusText?: string;
```

Comment:
Removing these makes this a breaking change :(

Reply:
Whoops, never mind - didn't realise they were getters, not methods :)

Reply:
Although then - shouldn't `reponse` be a getter as well?

Reply:
Ah right, it's just to read the property from `reponse`, not to actually make it readonly. So much for my conversation with myself ;)

Anyway, if you do want to deprecate these in favour of reading them from the response object, the `@deprecated` TSDoc annotation will mark it as such (preferably combined with a comment describing its replacement).

Reply:
Agreed - though if we do deprecate it, I'd prefer to mark it deprecated at the next minor release. It's a handy enough shortcut that maybe it's worth discussion.